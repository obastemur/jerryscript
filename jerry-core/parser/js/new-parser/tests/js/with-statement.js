/* This test contains with statement parser tests. */

function func(arg,val,c,arg,val)
{
  with (val) {
    v1 = arg;
    {
      try {
        with (ident)
          arg.other = v1.val;
      } finally {}
    }
    arg.other++
  }

  var v1, ident, other;
  other = val;
}
