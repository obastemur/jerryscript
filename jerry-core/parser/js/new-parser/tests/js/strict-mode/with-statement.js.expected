
--- Script parsing start ---

  Note: switch to strict mode


--- Script parsing end ---

Parse error 'With statement not allowed in strict mode.' at line: 5 col: 1
